arbeitsplan/templates/home.html

```diff
@@ -12,191 +12,243 @@
+      <i class="bi fa-solid fa-eye fa-lg" width="1em" height="1em"></i>
+    </div>
+    <div>
+      <a href="/arbeitsplan/leistungListe/" class="icon-link">Leistungsübersicht</a>
+      <p> Eine Übersicht meiner geleistenen Arbeiten anzeigen.</p>
+    </div>
+  </div>
+  <div class="col d-flex align-items-start">
+    <div
+      class="icon-square text-body-emphasis bg-body-secondary d-inline-flex align-items-center justify-content-center fs-4 flex-shrink-0 me-3">
+      <i class="bi fa-solid fa-file-pen fa-lg" width="1em" height="1em"></i>
+    </div>
+    <div>
+      <a href="/arbeitsplan/leistungAlle/" class="icon-link">Leistung eintragen (beliebige)</a>
+      <p>Geleisteten Arbeitsdienst für beliebige Aufgabe eintragen.</p>
     </div>
   </div>
-
 </div>
 
-
 {% if user.teamleader_set.count > 0 %}
-<h2> Teamleiter-Funktionen </h2>
-<p> 
-
-<div class="row row-cols-1 row-cols-lg-2 g-4">
 
-  <div class="col">        
-    <div class="card">
-      <div class="card-header text-bg-secondary text-center">
-        <i class="fa-solid fa-list-ul fa-lg fa-fw"></i>&nbsp;Aufgaben
-      </div>
-      <ul class="list-group list-group-flush">
-        <li class="list-group-item"><a href="{% url 'arbeitsplan-aufgabenTeamleader' %}">Für welche Aufgaben bin ich Teamleiter? </a> </li>
-      </ul>
+<h2 class="pb-2 border-bottom"></h2>
+<h2> Teamleiter-Funktionen </h2>
+<p>
+<div class="row g-4 py-5 row-cols-1 row-cols-lg-2">
+  <div class="col d-flex align-items-start">
+    <div
+      class="icon-square text-body-emphasis bg-body-secondary d-inline-flex align-items-center justify-content-center fs-4 flex-shrink-0 me-3">
+      <i class="bi fa-solid fa-people-group fa-lg" width="1em" height="1em"></i>
+    </div>
+    <div>
+      <a href="{% url 'arbeitsplan-aufgabenTeamleader' %}" class="icon-link">Aufgaben anzeigen</a>
+      <p>Für welche Aufgaben bin ich Teamleiter?</p>
     </div>
   </div>
-
-  <div class="col">        
-    <div class="card">
-      <div class="card-header text-bg-secondary text-center">
-        <i class="fa-solid fa-check fa-lg fa-fw"></i>&nbsp;Leistung
-      </div>
-      <ul class="list-group list-group-flush">
-        <li class="list-group-item"><a href="/arbeitsplan/leistungenBearbeiten/z=team/">Vorliegende Leistungsmeldungen für mein Team bearbeiten </a> </li>
-      </ul>
+  <div class="col d-flex align-items-start">
+    <div
+      class="icon-square text-body-emphasis bg-body-secondary d-inline-flex align-items-center justify-content-center fs-4 flex-shrink-0 me-3">
+      <i class="bi fa-solid fa-check fa-lg" width="1em" height="1em"></i>
+    </div>
+    <div>
+      <a href="/arbeitsplan/leistungenBearbeiten/z=team/" class="icon-link">Leistungsmeldungen anzeigen</a>
+      <p>Vorliegende Leistungsmeldungen für mein Team bearbeiten.</p>
     </div>
   </div>
-
 </div>
 {% endif %}
 
 
 {% for g in user.groups.all %}
 {% if g.name == "Vorstand" %}
+<h2 class="pb-2 border-bottom"></h2>
```

Comment:
Better use a horizontal ruler `<hr>`. Taking an empty heading an underlining it seems like the Word approach.